src/main/scala/org/constellation/snapshotstreaming/FileBasedLastIncrementalGlobalSnapshotStorage.scala

```diff
@@ -29,15 +29,17 @@ import io.circe.syntax._
 
 object FileBasedLastIncrementalGlobalSnapshotStorage {
 
-  def make[F[_]: Async: Files: KryoSerializer](
-    path: Path
+  def make[F[_]: Async: Files: KryoSerializer: Hasher](
+    path: Path,
+    hashSelect: HashSelect
   ): F[LastSnapshotStorage[F, GlobalIncrementalSnapshot, GlobalSnapshotInfo]] =
     Semaphore[F](1)
-      .map(make(path, _))
+      .map(make(path, _, hashSelect))
 
-  private def make[F[_]: Async: Files: KryoSerializer](
+  private def make[F[_]: Async: Files: KryoSerializer: Hasher](
```

Comment:
Is `KryoSerializer` still needed here given you added Hasher?

Reply:
No idea. Probably I'd have to change the build parameters as you did in tessellation, to see that. Right?

Reply:
Or just try removing it and see if it compiles.

Reply:
Ok, I'll try that in a separate PR